### PR TITLE
core: tools: bootstraps: Add missing 'set -e' to raise if any command fails

### PR DESCRIPTION
**Pull request by patrickelectric**
I got a situation where ttyd fails to download but the ttyd file is there and empty

### CODE REVIEW

File: core/tools/ardupilot_tools/bootstrap.sh [hunk 1]
```diff
@@ -9,8 +9,13 @@ set -e
 COMMIT_HASH=4ea8c32c61781fa36dff9748fe3a18cdb5743abb
 LOCAL_PATH_UPLOADER="/usr/bin/ardupilot_fw_uploader.py"
 REMOTE_URL_UPLOADER="https://raw.githubusercontent.com/ArduPilot/ardupilot/${COMMIT_HASH}/Tools/scripts/uploader.py"
-wget "$REMOTE_URL_UPLOADER" -O "$LOCAL_PATH_UPLOADER"
-chmod +x "$LOCAL_PATH_UPLOADER"
+
+# Sudo command is not available on docker and the script is also used in different environmensts
```
Comment (Williangalvani) [inline core/tools/ardupilot_tools/bootstrap.sh:13]: environments*

File: core/tools/ardupilot_tools/bootstrap.sh [hunk 2]
```diff
@@ -9,8 +9,13 @@ set -e
...
+# The SUDO lias allow the usage of sudo when such command exists and also ignore if it doesn't
```
Comment (Williangalvani) [inline core/tools/ardupilot_tools/bootstrap.sh:14]: alias*

File: core/tools/ardupilot_tools/bootstrap.sh (hunk 1 above)
Comment (patrickelectric) [reply core/tools/ardupilot_tools/bootstrap.sh]: Fixed.

File: core/tools/ardupilot_tools/bootstrap.sh (hunk 2 above)
Comment (patrickelectric) [reply core/tools/ardupilot_tools/bootstrap.sh]: Fixed.

**Pull request merged**